crossfire/__init__.py

```diff
@@ -1,9 +1,9 @@
 __version__ = "0.1.0"
-__all__ = ("cities", "ocurrences", "states")
+__all__ = ("cities", "occurrences", "states")
```

Comment:
Aqui, se não em engano, a gente tem que incluir `"Client"` and `"AsyncClient"` — como já estava assim, podemos fazer em outra issue/PR : )